### ISSUE TITLE
Three OCR errors in the file 汉字表 2023 （only 3000).csv

### ISSUE THREAD

**Issue by becky82** [shawkynasr/HSK-official-Query-System#9]
I noticed three OCR errors in levels 7-9 relevant for the file: 汉字表 2023 （only 3000).csv

- 脏 (a HSK2 character) should be 赃 [page 29, character 1101 in the Standards]
- 洵 should be 汛 [page 29, character 1000]
- 箫 should be 萧 [page 28, character 957]

(I think these have already  been fixed in [汉字.csv](https://github.com/shawkynasr/HSK-official-Query-System/blob/main/%E6%B1%89%E5%AD%97.csv))

**Comment by shawkynasr** [comment]
I will check it, thanks!

**Comment by shawkynasr** [comment]
You are right [汉字.csv](https://github.com/shawkynasr/HSK-official-Query-System/blob/main/%E6%B1%89%E5%AD%97.csv)) more accurate. I will remove 2023 file.
